pages/1.13/security/oss/user-account-management/index.md

```diff
@@ -0,0 +1,21 @@
+---
+layout: layout.pug
+navigationTitle:  User Account Management
+title: User Account Management
+menuWeight: 10
+excerpt: Managing DC/OS user accounts
+---
+
+<!-- The source repository for this topic is https://github.com/dcos/dcos-docs-site -->
+
+DC/OS is designed as a multi-user system.
+
+Open DC/OS can handle three types of user accounts:
+
+* **External user accounts**: External user accounts exist for users who want to use their Google, GitHub, or Microsoft credentials. DC/OS does never receive or store the passwords of external users.
```

Comment:
What do you think about "External users log in via single sign-on through Auth0 (using their Google, GitHub, or Microsoft credentials)"?